src/pages/products/sgid/society/state-agency-dispatch.astro

```diff
@@ -27,7 +27,7 @@ export const metadata: IMetadata = {
 
 const page: IPageMetadata = {
   ...metadata,
-  updateHistory: [`October 2013`],
```

Comment:
Please keep all the history on product pages. Add back the removed date and we'll get this merged.

Reply:
Added back!